test/powershell/Modules/Microsoft.PowerShell.Management/Join-Path.Tests.ps1

```diff
@@ -50,4 +50,38 @@ Describe "Join-Path cmdlet tests" -Tags "CI" {
     $result.Count | Should -Be 1
     $result       | Should -BeExactly "one${sepChar}two${sepChar}three${sepChar}four${sepChar}five"
   }
+  It "Join-Path -Path <Path> -ChildPath <ChildPath> should return '<ExpectedResult>'" -TestCases @(
+    @{
+        Path = 'one'
+        ChildPath = 'two', 'three'
+        ExpectedResult = "one${sepChar}two${sepChar}three"
+    }
+    @{
+        Path = 'one', 'two'
+        ChildPath = 'three', 'four'
+        ExpectedResult = @(
+            "one${sepChar}three${sepChar}four"
+            "two${sepChar}three${sepChar}four"
+        )
+    }
+    @{
+        Path = 'one'
+        ChildPath = @()
+        ExpectedResult = "one${sepChar}"
+    }
+    @{
+        Path = 'one'
+        ChildPath = $null
+        ExpectedResult = "one${sepChar}"
+    }
+    @{
+        Path = 'one'
+        ChildPath = [string]::Empty
```

Comment:
We need one more test for empty collection.

Reply:
This should already be covered here: https://github.com/PowerShell/PowerShell/blob/0efc23fe6dcd3cd9cedc952f3bc9c580486a0607/test/powershell/Modules/Microsoft.PowerShell.Management/Join-Path.Tests.ps1#L67-L71